src/bin/gpfusion/gpbridgeapi.c

```diff
@@ -508,17 +531,14 @@ size_t fill_buffer(gphadoop_context* context, char* start, size_t size)
 void add_delegation_token(PxfInputData *inputData)
```

Comment:
can we just pass inputData->gphduri->dfs_address ? instead of the entire object?